<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>WdgLayerProperties</class>
 <widget class="QWidget" name="WdgLayerProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>224</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>&amp;Name:</string>
     </property>
     <property name="buddy" >
      <cstring>editName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="SqueezedComboBox" name="cmbProfile" />
   </item>
   <item rowspan="2" row="1" column="1" >
    <widget class="KisCmbIDList" name="cmbColorSpaces" />
   </item>
   <item row="5" column="1" >
    <widget class="KisCmbComposite" name="cmbComposite" />
   </item>
   <item row="4" column="1" >
    <widget class="KisIntSpinbox" name="intOpacity" />
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="lblChannelsst" >
     <property name="text" >
      <string>Visible channels:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy" >
      <cstring>lstChannels</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QListWidget" name="listChannels" />
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="editName" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="textLabel4" >
     <property name="text" >
      <string>Composite mode:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="textLabel3" >
     <property name="text" >
      <string>&amp;Opacity:</string>
     </property>
     <property name="buddy" >
      <cstring>intOpacity</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>Colorspace:</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="2" column="0" >
    <widget class="QLabel" name="textLabel1_2" >
     <property name="text" >
      <string>Profile:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KisIntSpinbox</class>
   <extends>QWidget</extends>
   <header>kis_int_spinbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>SqueezedComboBox</class>
   <extends>QWidget</extends>
   <header>squeezedcombobox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QWidget</extends>
   <header>kis_cmb_idlist.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KisCmbComposite</class>
   <extends>QWidget</extends>
   <header>kis_cmb_composite.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
